apps/web/pages/apps/categories/[category].tsx

```diff
@@ -47,13 +47,7 @@ export default function Apps({ apps }: InferGetStaticPropsType<typeof getStaticP
 }
 
 export const getStaticPaths = async () => {
-  const appStore = await getAppRegistry();
-  const paths = appStore.reduce((categories, app) => {
-    if (!categories.includes(app.category)) {
-      categories.push(app.category);
-    }
-    return categories;
-  }, [] as string[]);
```

Comment:
Since we're just getting the app categories we don't need to loop through all apps, we can just use the enum from the schema. This will also help with performance.